src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php

```diff
@@ -20,9 +20,15 @@
  */
 class TreeBuilder implements NodeParentInterface
 {
+    /**
+     * @var NodeInterface|null
+     */
     protected $tree;
+
+    /**
+     * @var NodeDefinition|null
+     */
     protected $root;
-    protected $builder;
```

Comment:
this should be deprecated instead with an annotation (no need for runtime BC layer since this is useless, WDYT?)